zerver/views/auth.py

```diff
@@ -89,8 +89,7 @@ def maybe_send_to_registration(request: HttpRequest, email: Text, full_name: Tex
         if multiuse_object_key is not None:
             del request.session["multiuse_object_key"]
             request.session.modified = True
-            if streams_to_subscribe is not None:
-                prereg_user.streams = streams_to_subscribe
+            prereg_user.streams = streams_to_subscribe
```

Comment:
@hackerkid is this the desired behavior for how `prereg_user.streams` works with multi-use invite links?  I had sorta imagined that `.streams` should be able to be None...